index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1 id="title">Generic Form</h1>
+    <p id="description">Short explanation...</p>
+
+    <form id="survey-form">
+      <label for="name-label" id="name-label">
+        Name
+        <input placeholder="name" type="text" id="name" required />
+      </label>
+
+      <label for="email-label" id="email-label">
+        email
```

Comment:
### Semantic tags
I think it would be better if you use the <span> tag on the text inside the label to make your code more semantic right.